core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/TermsAggregatorBuilder.java

```diff
@@ -71,6 +65,45 @@ public TermsAggregatorBuilder(String name, ValueType valueType) {
         super(name, StringTerms.TYPE, ValuesSourceType.ANY, valueType);
     }
 
+    /**
+     * Read from a stream.
+     */
+    public TermsAggregatorBuilder(StreamInput in) throws IOException {
+        super(in, StringTerms.TYPE, ValuesSourceType.ANY);
+        bucketCountThresholds = BucketCountThresholds.readFromStream(in);
+        collectMode = SubAggCollectionMode.BREADTH_FIRST.readFrom(in);
+        executionHint = in.readOptionalString();
+        if (in.readBoolean()) {
+            includeExclude = IncludeExclude.readFromStream(in);
+        }
+        order = InternalOrder.Streams.readOrder(in);
+        showTermDocCountError = in.readBoolean();
+    }
+
+    @Override
+    protected boolean serializeTargetValueType() {
```

Comment:
nit: can we move this method down below `innerWriteTo()` so the read and write methods are next to each other?